Plugins/GitUIPluginInterfaces/IGitModule.cs

```diff
@@ -128,6 +128,9 @@ public interface IGitModule
         string GetEffectiveSetting(string setting);
         string GetEffectiveGitSetting(string setting, bool cache = true);
 
+        /// <summary>Gets the current branch; or, if HEAD is detached, null if setDefaultIfEmpty is true and "(no branch)" otherwise.</summary>
```

Comment:
Please rewrite as a correct xml-doc, i.e.:
* "summary" provides an overview of the API.
* "returns" describes the return values (e.g., does it return a commit hash or a branch name?)

As well as the parameter description.